src/framegrab/cli/autodiscover.py

```diff
@@ -5,21 +5,33 @@
 from imgcat import imgcat
 
 from framegrab import FrameGrabber
-from framegrab.cli.clitools import PREVIEW_COMMAND_CHOICES, preview_image
+from framegrab.cli.clitools import (
+    PREVIEW_COMMAND_CHOICES,
+    PREVIEW_RTSP_COMMAND_CHOICES,
+    preview_image,
+)
+from framegrab.rtsp_discovery import AutodiscoverModes
 
 
 @click.command()
-@click.argument(
-    "preview",
+@click.option(
+    "--preview",
     type=click.Choice(PREVIEW_COMMAND_CHOICES, case_sensitive=False),
     default="imgcat",
+    show_default=True,
+)
+@click.option(
+    "--rtsp_discover_modes",
+    type=click.Choice(PREVIEW_RTSP_COMMAND_CHOICES, case_sensitive=False),
+    default="light",
+    show_default=True,
 )
-def autodiscover(preview: str):
+def autodiscover(preview: str, rtsp_discover_modes: str):
```

Comment:
This technically would make this a breaking change. Can we add a default to keep it compatible?